### PR TITLE
V: fix dockerfile

### PR DESCRIPTION
**Pull request by enghitalo**

### CODE REVIEW

File: v/Dockerfile [hunk 1]
```diff
@@ -12,7 +13,7 @@ WORKDIR /app
 
 RUN v . -prod -o server
 
-FROM alpine
+FROM alpine:latest
```
Comment (waghanza) [inline v/Dockerfile:16]: not better to be `debian` ?

File: v/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,7 @@
 FROM thevlang/vlang:debian as build
 
 WORKDIR /app
+RUN v up
```
Comment (waghanza) [inline v/Dockerfile:4]: If I understand correctly, `v up` fetch the latest version. Then the version displayed on `config.yaml` will be wrong

**Review by waghanza — CHANGES_REQUESTED** [review]

File: v/Dockerfile (hunk 2 above)
Comment (enghitalo) [reply v/Dockerfile:4]: v still on V 0.4.x

Comment (waghanza) [reply v/Dockerfile:4]: for noz, but in the future `v up` could lead to 0.5 right ?

is there any param like `v up -v 0.4` ?

File: v/Dockerfile [hunk 3]
```diff
@@ -1,6 +1,7 @@
 FROM thevlang/vlang:debian as build
```
Comment (waghanza) [inline v/Dockerfile:1]: weird, this lead to alpine instead of debian
cc @koplenov

File: v/Dockerfile (hunk 2 above)
Comment (waghanza) [reply v/Dockerfile:4]: also should not be `v up -prod` ?

**Review by waghanza — CHANGES_REQUESTED** [review]

File: v/Dockerfile (hunk 2 above)
Comment (enghitalo) [reply v/Dockerfile:4]: > for noz, but in the future `v up` could lead to 0.5 right ?
> 
> is there any param like `v up -v 0.4` ?

I can change to `latest`. What do you think?

Comment (enghitalo) [reply v/Dockerfile:4]: > also should not be `v up -prod` ?

No

File: v/Dockerfile (hunk 1 above)
Comment (enghitalo) [reply v/Dockerfile:16]: I think not

**Review by waghanza — CHANGES_REQUESTED** [review]
I have 
```v
V panic: [VEX] Failed to listen to port 3000
v hash: ca3da8b
backtrace_symbols is missing => printing backtraces is not available.
Some libc implementations like musl simply do not provide it.
```
when running the produced binary

**Comment by enghitalo** [comment]
> I have
> 
> ```v
> V panic: [VEX] Failed to listen to port 3000
> v hash: ca3da8b
> backtrace_symbols is missing => printing backtraces is not available.
> Some libc implementations like musl simply do not provide it.
> ```
> 
> when running the produced binary

fixed on last commit

**Comment by enghitalo** [comment]
@waghanza CI was stopped?

**Comment by waghanza** [comment]
I h

> @waghanza Was CI stopped?

No. I have to approve since your are a first time contributor

**Comment by cyrusmsk** [comment]
> fixed on last commit

There is also a tricky part with vweb (or with x.vweb I don't remember exactly) - that it has nice performance only if it is built with "-gc none".. It is leaking a bit but it is fine)
But other frameworks don't need it..

**Comment by enghitalo** [comment]
@waghanza Can this workflow be rerun?

**Comment by waghanza** [comment]
time to continue @enghitalo @cyrusmsk @koplenov ?

**Comment by enghitalo** [comment]
> time to continue @enghitalo @cyrusmsk @koplenov ?

I will try finish it in next 15 days

File: v/Dockerfile [hunk 4]
```diff
@@ -10,9 +15,9 @@ WORKDIR /app
   RUN {{{.}}}
 {{/bootstrap}}
 
-RUN v . -prod -o server
+RUN v . {{#build_flags}} {{{.}}} {{/build_flags}} -o server
 
-FROM alpine
+FROM thevlang/vlang:debian-dev
```
Comment (waghanza) [inline v/Dockerfile:20]:
```suggestion
FROM debian
```

File: v/vex/config.yaml [hunk 5]
```diff
@@ -1,6 +1,10 @@
 framework:
   github: nedpals/vex
-  version: 0.3.5
+  version: 0.3.6
```
Comment (waghanza) [inline v/vex/config.yaml:3]:
```suggestion
  version: 0.3
```

**Review by waghanza — CHANGES_REQUESTED** [review]

**Comment by waghanza** [comment]
what is the release cycle of `v` @koplenov @enghitalo 

any reason forusing a 2023 version for pico and 2024 for vweb?

**Comment by enghitalo** [comment]
> what is the release cycle of `v` @koplenov @enghitalo

There is no standard but, at most, every month.

**Review by waghanza — APPROVED** [review]

**Pull request merged**